.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+## Description of Changes: 
+One or more paragraphs describing the problem, solution, and required changes.
+
+## Tests Conducted: 
+Explicitly state what tests were run on these changes, or if any are still pending (for README or other text-only changes, just put "None required". Make note of the compilers used, the platform/machine, and other relevant details as necessary. For more complicated changes, or those resulting in scientific changes, please be explicit!  
+**OR** Add any links to tests conducted. For example, "See ufs-community/ufs-weather-model/pull/<pr_number>"
+
+## Dependencies:
+Add any links to parent PRs (e.g. SCM and/or UFS PRs). For example:
+- NCAR/ccpp-physics/pull/<pr_number>
+- NOAA-EMC/fv3atm/pull/<pr_number>
+- ufs-community/ufs-weather-model/pull/<pr_number>
+
+## Documentation:
+Does this PR add new capabilities that need to be documented or require modifications to the existing documentation?  If so, brief supporting material can be provided here. Contact the CODEOWNERS if your PR requires extensive updates to the documentation.  See https://github.com/NCAR/ccpp-doc for Technical Documentation or https://dtcenter.org/community-code/common-community-physics-package-ccpp/documentation for the latest Scientific Documentation.
+
+## Issue (optional): 
```

Comment:
I would prefer that this is not optional. That is, I find it helps to always have at least one issue that has been entered to describe the work (e.g., bug report, feature request).